src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -203,6 +235,9 @@ IConfigurationSource configurationSource
     public class StateVectorToHtmlResultEncoder : IResultEncoder
     {
         private const double TWO_PI = 2.0 * System.Math.PI;
+        private double count = 0.0;
```

Comment:
I'd strongly suggest making this an int rather than a double, since it's only being used to hold integer values.
```suggestion
        private int count = 0;
```